### PR TITLE
feat: maintainer badge

### PR DESCRIPTION
**Pull request by cloudhead**
2020-08-25-184130_352x210_scrot
2020-08-25-184138_532x400_scrot

This adds the maintainer badge to maintainers in the revision selector and the projects page.

### CODE REVIEW

**Review by xla — DISMISSED** [review]
🌵 ♌️ 🌊 🐏

**Review by rudolfs — CHANGES_REQUESTED** [review]
Looks like you forgot to add:

```
import { getContext } from "svelte";
const { registrationFee } = getContext("session");
```

in `Project.svelte`.

Screenshot 2020-08-26 at 09 48 05

That's also why the build is failing.

**Comment by xla** [comment]
Looks like merge snafu from #817.

**Review by rudolfs — APPROVED** [review]

File: ui/DesignSystem/Component/Badge.svelte [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<script>
+  export let variant = null;
+  export let style = null;
+
+  const caption = String(variant);
+</script>
+
+<style>
+  .badge {
+    color: var(--color-primary);
```
Comment (brandonhaslegs) [inline ui/DesignSystem/Component/Badge.svelte:10]: The color should be `--color-secondary` and background should be `--color-secondary-level-1` according to Figma.
image

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]

File: ui/DesignSystem/Component/Badge.svelte [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    background: var(--color-primary-level-1);
+    border-radius: 4px;
+    padding: 0.125rem 0.25rem;
```
Comment (brandonhaslegs) [inline ui/DesignSystem/Component/Badge.svelte:13]: The height of the badge should be 24px. It's currently 21.
image

**Review by rudolfs — APPROVED** [review]

**Review by brandonhaslegs — APPROVED** [review]

**Pull request merged**